ci/bundle_custom_data.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+exclude_verify_shape_list = []
+
+exclude_verify_torchscript_list = ["swin_unetr_btcv_segmentation"]
+
+# please add the bundle (as a key) and a dict in the form of {"net_id": "", "config_file": ""} (as a value)
```

Comment:
Could you please help add more description about what these `lists` and `dicts` in this file are used for?

Thanks.